templates/default/serverspec.erb

```diff
@@ -0,0 +1,7 @@
+require 'serverspec'
+
+set :backend, :exec
+
+describe file('/') do
```

Comment:
There probably doesn't need to be a test in here. I'll check if test kitchen complains without tests, and remove it if it doesn't matter.